.github/workflows/client-celo.yml

```diff
@@ -0,0 +1,79 @@
+name: Go / Celo
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-celo-cache
+          key: ${{ runner.os }}-buildx-celo-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-celo-
+      # TODO: This step was left here intentionally so we can track disk space
+      # usage for a while. We were trying to fight problems with out of disk space
+      # that happened due to the size of data restored from cache. The cache size
+      # was growing linearly with subsequent workflow runs. We want to observe
+      # available disk space for `/`. Fresh execution starts with 21 GB, we expect
+      # to have no less than 15 GB after the cache is restored.
+      - run: sudo df -h
+
+      - name: Build Docker Build Image
+        uses: docker/build-push-action@v2
+        with:
+          target: gobuild
+          tags: go-build-env-celo
+          build-args: |
+            HOST_CHAIN=celo
+          # REVISION=${{ github.sha }} TODO: set revision only when we want to publish the image
+          # VERSION= ? TODO: Configure version, sample: 1.7.6
+          load: true # load image to local registry to use it in next steps
+          cache-from: type=local,src=/tmp/.buildx-celo-cache
+          cache-to: type=local,dest=/tmp/.buildx-celo-cache
+
+      - name: Create test results directory
+        run: mkdir test-results-celo
+
+      - name: Run Go tests
+        run: |
+          docker run \
+            --volume $GITHUB_WORKSPACE/test-results-celo:/mnt/test-results-celo \
+            --workdir /go/src/github.com/keep-network/keep-ecdsa \
+            go-build-env-celo \
+            gotestsum --junitfile /mnt/test-results-celo/unit-tests-celo.xml -- -tags=celo,musl ./...
```

Comment:
I'm not 100% sure, but `./...` looks to me as unnecessary. We already provide the path specifying `/mnt/test-results-celo/unit-tests-celo.xml`.

Reply:
It's not set the path of test results. That tells the `gotestsum` where to find test files. This is needed because we use double dash (`--`) to set build tags so positional arguments are taken into account by the `gotestsum` command and the default test command is not used in that case. Without that no tests are invoked.

Reply:
Oh, I see, thanks for the explanation